lib/krikri/enrichments/limit_characters.rb

```diff
@@ -0,0 +1,17 @@
+module Krikri::Enrichments
+  ##
+  # Enrichment to limit the number of characters in a string to 1000
+  #
```

Comment:
I know most of the enrichments don't do this, but I've been trying to get better at using YARD...
...maybe an `@example` tag, here?